charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
```

Comment:
Use `{{- /*` here to wipe out any previous newlines (line 30) in the final rendered template.